packages/web3-utils/test/fixtures/stringManipulation.ts

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable @typescript-eslint/no-magic-numbers */
+
+import { HexString, Numbers } from "../../src/types";
+
+
+export const padLeftData: [[Numbers, number, string], HexString][] = [
+    [[0, 10, '0'], '0x0000000000'],
+    [['0x01', 64, 'f'], '0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff01'],
+    [['-0x01', 64, 'f'], '-0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff01'],
+    [['5', 32, '0'], '00000000000000000000000000000005'],
+    [['-05', 32, 'f'],'fffffffffffffffffffffffffffff-05'],
+    [['abcd', 8, '0'],'0000abcd'],
+    [['-abcd', 8, '0'],'000-abcd'],
+    [[BigInt('9007199254740992'),32,'0'],'0x00000000000000000020000000000000'],
```

Comment:
```suggestion
    [[BigInt('9007199254740992'),32,'0'],'0x00000000000000000020000000000000'],
    [[9007199254740992n,32,'0'],'0x00000000000000000020000000000000'],
```

Add test case for alternative syntax for BigInt